docs/resources/USER.md

```diff
@@ -67,6 +67,20 @@ A brief version of a [guild](#DOCS_GUILD/guild-object) object
 }
 ```
 
+## Connection Object
+
+The connection object that the user has attached.
+
+###### Connection Structure
+
+| Field | Type | Description |
+|-------|------|-------------|
+| id | string | id of the connection account |
+| name | string | the username of the connection account |
+| type | string | the service of the connection (twitch, youtube) |
+| revoked | bool | whether the connection is revoked |
+| integrations | array | an array of integrations |
```

Comment:
it returns partial server [integration objects](https://github.com/hammerandchisel/discord-api-docs/blob/2d557053cab1c181bf58f9f5c1e32572f8c40f77/docs/resources/GUILD.md#integration-object)